commands/login_option

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+
+help() {
+  cat <<EOF
+----------------------------------------------------------------------------------------------------------
+HELP MESSAGE:
+    Usage: c8y bpl $(basename "$0") <command> <argument>
+    Commands:
+      remove_restriction        Remove login authentication restriction
+      help                      Show this help message and exit
+    Arguments:
+      -h, --help                Show this help message and exit
+    Example usage:
+    Remove authentication login restriction:
+        c8y bpl login_option remove_restriction
+----------------------------------------------------------------------------------------------------------
+EOF
+}
+
+extract_id_and_type() {
+    local json_string=$1
+
+    # Remove leading/trailing whitespace
+    local cleaned_string=${json_string#"{"}
+    # Remove curly braces
+    cleaned_string=${cleaned_string%"}"}
+
+    # Extract the "id" value
+    local id=$(grep -o '"id":"[^"]*' <<< "$cleaned_string" | cut -d":" -f2 | tr -d '"')
+
+    # Extract the "providername" value
+    local providerName=$(grep -o '"providername":"[^"]*' <<< "$cleaned_string" | cut -d":" -f2 | tr -d '"')
+
+    # Create an array and add the values
+    local result=()
+    result+=("$id")
+    result+=("$providerName")
+
+    # Return the array
+    echo "${result[@]}"
+}
+
+remove_restriction() {
+  idAndProviderName=$(c8y api -n get $C8Y_HOST/tenant/loginOptions --filter "type like BASIC" --select "id,providerName" "$@")
+  echo "idAndProviderName (should look something like '8caad89a-e1a2-4d28-8d31-ae194d3f1dbb,'): $idAndProviderName"
+  output=$(extract_id_and_type "$idAndProviderName")
+  # Extract the values from the array and remove whitespace
+  id=$(echo "${output[0]}" | tr -d ' ')
+  providerName=$(echo "${output[1]}" | tr -d ' ')
+
+  # Print the extracted values individually
+  echo "ID: $id" >&2
+  echo "Provider Name: $providerName" >&2
+
+  response=$(c8y api -n put $C8Y_HOST/tenant/loginOptions/$id --debug --template "{type: 'BASIC', 'providerName': \"$providerName\", 'id': \"$id\", 'authenticationRestrictions': {'forbiddenClients': []}}" "$@")
```

Comment:
--debug should not be used by default but only if the user adds it to the initial command